packages/dcos-integration-test/extra/test_cockroach_config.py

```diff
@@ -0,0 +1,28 @@
+"""
+Testing the CockroachDB setup in DC/OS.
+"""
+import logging
+import subprocess
+from time import sleep
+
+
+__maintainer__ = 'tweidner'
+__contact__ = 'security-team@mesosphere.io'
+
+
+LOG = logging.getLogger(__name__)
+
+
+def test_config_change_applied():
+    subprocess.check_call(
+        ['sudo', 'systemctl', 'restart', 'dcos-cockroachdb-config-change'],
+    )
+    # Wait a few moments for the CRDB config change script to finish.
+    sleep(3)
```

Comment:
Is there anything we can poll for, here? Rather than sleep. Not a blocker, but it's not uncommon for systemd queues, io, etc make the execution time much less deterministic.